improver_tests/acceptance/test_threshold_interpolation.py

```diff
@@ -0,0 +1,48 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Test for the threshold interpolation CLI."""
+
+import pytest
+
+from . import acceptance as acc
+
+pytestmark = [pytest.mark.acc, acc.skip_if_kgo_missing]
+CLI = acc.cli_name_with_dashes(__file__)
+run_cli = acc.run_cli(CLI)
+
+
+@pytest.mark.parametrize("with_new_thresholds", (True, False))
+def test_basic(tmp_path, with_new_thresholds):
+    """Test basic invocation with and without threshold argument"""
+    thresholds = (
+        "50.0,100.0,150.0,200.0,250.0,300.0,350.0,400.0,450.0,500.0,600.0,"
+        "700.0,800.0,900.0,1000.0,1500.0,2000.0,2500.0,3000.0,3500.0,4000.0,"
+        "4500.0,5000.0,5500.0,6000.0,6500.0,7000.0,7500.0,8000.0,8500.0,9000.0,"
+        "9500.0,10000.0,12000.0,14000.0,16000.0,18000.0,20000.0,25000.0,30000.0,"
+        "35000.0,40000.0,45000.0,50000.0"
```

Comment:
Seems like an excessive number of thresholds for testing, see comments on acceptance test data PR.